spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfiguration.java

```diff
@@ -65,6 +66,8 @@
 
 	private final Credentials credentials;
 
+	private final Optional<String> emulatorHost;
```

Comment:
I think going with the plain old nullable `String` instead of `Optional` will make things less verbose.

Reply:
Done.